packages/module/src/FileDropZone/FileDropZone.scss

```diff
@@ -1,5 +1,5 @@
 .pf-chatbot__dropzone {
-  height: 100vh;
+  height: 70vh;
```

Comment:
This is the size of the container; adjusting here mostly just so the example won't be tiny.